_includes/markdown/Vue.md

```diff
@@ -0,0 +1,110 @@
+The purpose of this document is to help guide you through working with a potential project using the JavaScript framework, Vue. Outside of The [Official Vue Documentation](https://vuejs.org/), you should find here personalized recommendations for using the framework based on the types of projects we typically see at 10up. We have broken these recommendations out into common elements we tend to interact with often in client work. If something you’re looking for isn’t represented you can either submit a pull request to this repository or refer to the [Official Vue Documentation](https://vuejs.org/v2/guide/).
+
+For general coding style recommendations please see the official [Vue style guide](https://vuejs.org/v2/style-guide/).
+
+## Figuring Out If Vue Is Right For Your Project
+
+Using the right tool for the job is critical in navigating a successful project. Will Vue always be the answer? Of course not. But there are some instances where you may want to use this framework over something like (for example) a collection of plugins, custom JavaScript, or another framework.
+
+Some questions you may want to ask yourself:
+* Will this project contain a lot of components that need to interact with an API?
+* Are you replicating a lot features already present in Vue like client side templating and state management?
+* Will the addition of a framework save time and performance versus creating a custom solution? 
+* Do your client side templates need to be accessible to a back end technology (like PHP in a WordPress template)?
+
+If you find yourself answering, “Yes,” to those questions, Vue may be a great fit for your project! If not, a more traditional WordPress theme build may be the best.
+
+## Implementing Vue on a Project
+
+There are several ways to include Vue.js in your project. From the inclusion of [Vue in your HTML page](https://vuejs.org/v2/guide/installation.html#CDN) via a script tag, to [installing via NPM](https://vuejs.org/v2/guide/installation.html#NPM) and through the [Vue CLI](https://vuejs.org/v2/guide/installation.html#CLI). We will take a look at the different methods and explain when to use them.
+
+### Script Element
```

Comment:
I'm not sure how I feel about using the CDN. Isn't this just another DNS lookup to add to the list?

Reply:
Yeah, but it it really isn't any different than enqueuing a script. Ideally we'll include the script with NPM, but on the off case we can't I think using the CDN is fine (at least for the purpose of this documentation). I think there are enough cases where we'll use Vue for one specific feature where it doesn't need to be rolled into the main JS entry point that it's worth mentioning this as an option.